tests/torch/quantization/test_strip.py

```diff
@@ -325,3 +326,41 @@ def test_nncf_strip_api(strip_type, do_copy):
 
     assert isinstance(strip_model.conv.get_pre_op("0").op, FakeQuantize)
     assert isinstance(strip_model.nncf.external_quantizers["/nncf_model_input_0|OUTPUT"], FakeQuantize)
+
+
+@pytest.mark.parametrize(
+    ("mode", "torch_dtype", "atol"),
+    (
+        (nncf.CompressWeightsMode.INT4_ASYM, torch.float32, 0.01),
+        (nncf.CompressWeightsMode.INT4_ASYM, torch.float16, 0.01),
+        (nncf.CompressWeightsMode.INT4_ASYM, torch.bfloat16, 0.01),
+        (nncf.CompressWeightsMode.INT4_SYM, torch.float32, 0.01),
+        (nncf.CompressWeightsMode.INT4_SYM, torch.float16, 0.01),
+        (nncf.CompressWeightsMode.INT4_SYM, torch.bfloat16, 0.01),
```

Comment:
Is it possible to reduce some thresholds for `sym` or `float32`?

Reply:
Done